tests/instrumentation/asyncio/asyncpg_tests.py

```diff
@@ -0,0 +1,116 @@
+#  BSD 3-Clause License
+#
+#  Copyright (c) 2019, Elasticsearch BV
+#  All rights reserved.
+#
+#  Redistribution and use in source and binary forms, with or without
+#  modification, are permitted provided that the following conditions are met:
+#
+#  * Redistributions of source code must retain the above copyright notice, this
+#    list of conditions and the following disclaimer.
+#
+#  * Redistributions in binary form must reproduce the above copyright notice,
+#    this list of conditions and the following disclaimer in the documentation
+#    and/or other materials provided with the distribution.
+#
+#  * Neither the name of the copyright holder nor the names of its
+#    contributors may be used to endorse or promote products derived from
+#    this software without specific prior written permission.
+#
+#  THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+#  AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+#  IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+#  DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+#  FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+#  DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+#  SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+#  CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+#  OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+#  OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+import pytest  # isort:skip
+
+asyncpg = pytest.importorskip("asyncpg")  # isort:skip
+
+import os
+
+from elasticapm.conf import constants
+
+try:
+    from psycopg2 import sql
```

Comment:
there's no need for this import block, asyncpg does rely on psycopg2

Reply:
could you please elaborate on this one? how exactly does it rely on `psycopg2`? From what I see there are no dependencies on the mentioned package, e.g. [here](https://github.com/MagicStack/asyncpg/search?q=psycopg2&unscoped_q=psycopg2). 
Also I've used [pipgrip](https://github.com/ddelange/pipgrip) to check the full dependency tree and only found that `asyncpg` does not depend on other packages.

Reply:
hah 🤦 type, 
> there's no need for this import block, asyncpg does `not` rely on psycopg2

but yeah my point is: no need to import psycopg2 in this test. it's used in aiopg 1) because of the dependency and 2) because its used to prepare a statement, however asyncpg doesn't have such a concept

it's not a big deal either way, was just pointing out its not necessary here like it is in aiopg instrumentation tests

Reply:
oh, i see. that's a great explanation. thanks for that! I'll remove it.